docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -214,7 +184,17 @@ Verify the sync by checking the logs:
 
 ## Allowlist IP address
 
-You may need to allowlist one of our IP addresses to enable access to Airbyte:
-- 34.106.109.131
-- 34.106.196.165
-- 34.106.60.246
+Depending on your data residency location, you may need to allowlist the following IP addresses to enable access to Airbyte:
+
+### United States and Airbyte Default
+* 34.106.109.131
+* 34.106.196.165
+* 34.106.60.246
+
+### Europe
```

Comment:
We landed with "European Union" rather than "Europe" ([slack thread](https://airbytehq-team.slack.com/archives/C03A8CSANPQ/p1667399701612109))

```suggestion
### European Union
```